src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/Dispute.java

```diff
@@ -35,8 +36,8 @@ public class Dispute {
 
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
     @Id
-    @GeneratedValue (strategy = GenerationType.IDENTITY)
-    private Integer id;
+    @GeneratedValue
+    private UUID id;
```

Comment:
This will probably work for now, but I think when we actually use Oracle, the id will still be the primary key, but the new UUID will be an indexed unique key.